application/constants/templates.py

```diff
@@ -2,6 +2,10 @@
 Templates related constants.
 """
 from .base_enum import StrEnum
+from typing import Union
```

Comment:
Please sort import in right order. There is convention of sorting imports:

1. future imports `from __future__ import some_feature_that_supported_in_higher_python_version`(we don't have such here).
2. imports from standard library.
3. Third party imports Like: `from pydantic import Field`.
4. project imports. Like: `from service_hub import some_thing`(we don't have such example here)
5. local import like: `from .base_enum import StrEnum`.

Additional to it. In this project we have convention that one line one import.

Reply:
You can see import example here: https://github.com/JovianX/Service-Hub/blob/main/application/api/v1/schemas/applications.py